src/bibleBooks/fi.ts

```diff
@@ -0,0 +1,422 @@
+import type { BibleBook } from '@/types';
+
+export const bibleBooksFI: readonly BibleBook[] = [
+  {
+    id: 1,
+    prefix: '1',
+    aliases: ['moos', "mooses", "mo", "mooseksen"],
```

Comment:
aliases are only needed if one does not include the other. i.e. `moos` is included in `mooses`. But `sr` is not included in `saarnaaja`.
So in this case `['mooses', 'mooseksen']` is enough.
Also please lint the files to keep a constant codebase :)
`npm run lint:fix`